doc/source/ray-overview/getting-started.md

```diff
@@ -154,7 +107,7 @@ They are compatible with a variety of file formats, datasources, and distributed
 To get started with this example install Ray Data as follows.
 
 ```bash
-pip install "ray[data]" dask
+pip install -U "ray[default,data]" dask
```

Comment:
1. Technically we don't need default for these examples since they don't utitilize dashboard.
2. Probably a good idea to include required installs for the other libraries/sections as well!

Reply:
ack